code/modules/reagents/reagent_dispenser.dm

```diff
@@ -300,25 +300,28 @@
 
 //Beer Kegs
 
-/obj/structure/reagent_dispensers/beerkeg
-	name = "beer keg"
-	desc = "A beer keg"
+/obj/structure/reagent_dispensers/keg

```

Comment:
You will have to change the map stances of this item.

Reply:
Done.